lib/scss_lint/linter/use_variables_for_properties.rb

```diff
@@ -0,0 +1,18 @@
+module SCSSLint
+  class Linter::UseVariablesForProperties < Linter
+    include LinterRegistry
+
+    def visit_root(_node)
+      @included_properties = Set.new(config['included_properties'])
+      yield
```

Comment:
Since this linter only serves a purpose if there are properties defined, you can short circuit running anything at all by writing:

```ruby
yield if @properties.any?
```